pages/pages-root-folder/index.md

```diff
@@ -8,21 +8,22 @@ layout: frontpage
 header:
   image_fullwidth: caudri_challenge_banner_cropped_dark.jpg
 
-# widget1:
-#   title: "Blog & Portfolio"
-#   url: 'http://phlow.github.io/feeling-responsive/blog/'
-#   image: widget-1-302x182.jpg
-#   text: 'Every good portfolio website has a blog with fresh news, thoughts and develop&shy;ments of your activities. <em>Feeling Responsive</em> offers you a fully functional blog with an archive page to give readers a quick overview of all your posts.'
-# widget2:
-#   title: "Why use this theme?"
-#   url: 'http://phlow.github.io/feeling-responsive/info/'
-#   text: '<em>Feeling Responsive</em> is heavily customizable.<br/>1. Language-Support :)<br/>2. Optimized for speed and it&#39;s responsive.<br/>3. Built on <a href="http://foundation.zurb.com/">Foundation Framework</a>.<br/>4. Seven different Headers.<br/>5. Customizable navigation, footer,...'
-#   video: '<a href="#" data-reveal-id="videoModal"><img src="http://phlow.github.io/feeling-responsive/images/start-video-feeling-responsive-302x182.jpg" width="302" height="182" alt=""/></a>'
-# widget3:
-#   title: "Download Theme"
-#   url: 'https://github.com/Phlow/feeling-responsive'
-#   image: widget-github-303x182.jpg
-#   text: '<em>Feeling Responsive</em> is free and licensed under a MIT License. Make it your own and start building. The code is well-documented and explains you how it works.'
+widget1:
+  title: "Whats CAuDri?"
+  url: '/about/'
+  image: widget_teamphoto_302x182.webp
+  text: 'Who we are and what the CAuDri-Challenge is all about. Learn some of the history and future plans.'
+widget2:
+  title: "Get an impression!"
+  url: '/gallery/'
+  image: widget_gallery_302x182.webp
+  text: 'This is what it looked like at our previous CAuDri-Challenge events, what a blast!'
```

Comment:
"What a blast" always has this LLM vibe, but maybe that's just my opinion. 
What about "Take a look.." or "Get an impression of our previous events!"

Reply:
done